io/input/router.go

```diff
@@ -215,6 +215,8 @@ func (q *Router) Event(filters ...event.Filter) (event.Event, bool) {
 			t = f.Target
 		case transfer.TargetFilter:
 			t = f.Target
+		case transfer.URLFilter:
+			t = q // See comment in processEvent.
```

Comment:
Why not treat `transfer.URLFilter` the same as the tag-less `key.Filter` above, instead of inventing a fake tag?

Reply:
Because `q.key.scratchFilter = append(q.key.scratchFilter, f)` is a specific slice that holds key.Filter. Where the "generic one" (used by anything else) uses "taggedFilter". I don't think is right to add yet another specialised slice.